### PR TITLE
Aswathy/slider logic

### PR DESCRIPTION
**Pull request by aswathy-deriv**

### CODE REVIEW

File: src/components/tick-slider.jsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+          id={Styles.range}
+          min={day_mid()}
+          max={props.day_high}
+          class={Styles.slider2}
+          value={data().right}
+          step={1 / Math.pow(10, props.step_value)}
+        />
+        <div class={Styles.indicator}>
+          <div>
+            <div class={Styles.arrow_down} />
```
Comment (likhith-deriv) [inline src/components/tick-slider.jsx:51]: We have stylings for tick arrow in watchlist.module.scss, Can u see if u can reuse that? Or if this styling is better can u add it in shared, so it is consistant in the app

File: src/styles/slider.module.scss [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  margin-left: 10px;
+}
+
+.indicator {
+  display: flex;
+  justify-content: space-between;
+}
+
+.output_high {
+  padding-left: "5px";
```
Comment (likhith-deriv) [inline src/styles/slider.module.scss]: Can you remove the pixle measurement from quotes?

Comment (aswathy-deriv) [reply src/styles/slider.module.scss]: okay

**Review by vinu-deriv — APPROVED** [review]

File: src/components/tick-slider.jsx (hunk 1 above)
Comment (aswathy-deriv) [reply src/components/tick-slider.jsx:51]: we can check this only once the slider component is used inside any pages

**Pull request merged**